docker-compose.yml

```diff
@@ -28,8 +25,7 @@ services:
       replicas: 1
       restart_policy:
         condition: on-failure
-    ports:
-      - "8080:8080"
+    network_mode: host
     volumes:
        - ./config/cerberus.yaml:/root/cerberus/config/config.yaml:Z  # Modify the config in case of the need to monitor additional components
-       - /root/.kube/config:/root/.kube/config:Z
+       - ${HOME}/.kube/config:/root/.kube/config:Z
```

Comment:
Thinking if there's a way for user to pass the kubeconfig location when running the docker/podman-compose command like setting an env variable to avoid assuming a default location or having the user to tweak this file? Thoughts?

Reply:
Docker-compose supports env variables and possibly defaults, so yeah, sounds good.